src/hooks/index.js

```diff
@@ -44,11 +45,23 @@ exports.resolveRoleIds = function(app) {
 };
 
 function _resolveRoleId(app, name) {
-	const roleService = app.service('/roles');
-	return roleService.find({query: {name: name}})
-		.then(result => {
-			const role = result.data[0];
+	const roleModel = app.service('/roles').Model;
```

Comment:
I understand that sometime  you don't want hooks to run, but this is a hack and we shouldn't use hacks to make things working. There has to be another way.